docs/testlayer.rst

```diff
@@ -76,24 +76,29 @@ ZCMLLayer
 
 We now want a layer that loads up some ZCML from a file. The default
 is ``ftesting.zcml``, but here we'll load a test ``testlayer.zcml``.
+We can also choose to provide extra ZCML features (here, "devmode").
```

Comment:
There's documentation about features in general: http://zopeconfiguration.readthedocs.io/en/latest/narr.html#making-specific-directives-conditional

`devmode` is a feature that enables additional debugging features for developer convenience, like Django's `DEBUG = True`.  E.g. in my Zope 3 projects I used `hasFeature('devmode')` to decide whether to expose Python tracebacks to the end user in my SystemError views.